packages/amplication-git-pull-request-service/src/pull-request/dto/create-git-commit.dto.ts

```diff
@@ -3,7 +3,7 @@ import { IsOptional, IsString } from "class-validator";
 export class CreateGitCommit {
   @IsString()
   @IsOptional()
-  base?: string;
+  base?: string | undefined;
```

Comment:
Are you sure that the base could be `undefined`?
you must provide the base branch when you create a PR, and by default git takes the master/main branch

Reply:
Yes